kubeflow/hack/setup_iap_ingress.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# A helper script to create IAP ingress using the caller's GCP project.
+# The script will first create OAuth client and brand if these are
+# not provided.
+#
+# To find more information about the commands, please see
+# https://cloud.google.com/iap/docs/programmatic-oauth-clients.
```

Comment:
How do we provide redirect URI via command line approach? https://www.kubeflow.org/docs/distributions/gke/deploy/oauth-setup/

Reply:
Seems like redirect URIs are not supported in the commandline. Should we provide an option that skips this step if the user needs redirects?

Reply:
I think redirects is required for Kubeflow deployment to work, please correct me if I am wrong.

cc @gkcalat